packages/fluentui/react-northstar/src/components/Form/FormField.tsx

```diff
@@ -53,11 +54,23 @@ export interface FormFieldProps extends UIComponentProps, ChildrenComponentProps
 
   /** The HTML input type. */
   type?: string;
+
+  /** Message to be shown when input has error */
+  errorMessage?: ShorthandValue<TextProps>;
+
+  /** Indicator to be shown together with error message */
+  errorIndicator?: ShorthandValue<BoxProps>;
+
+  /** Indicator to be shown when field is required and non-empty */
+  satisfactoryIndicator?: ShorthandValue<BoxProps>;
```

Comment:
```suggestion
  successIndicator?: ShorthandValue<BoxProps>;
```
Could we call it `successIndicator` or does that not reflect the purpose of this indicator?

Reply:
I guess we can change, any objections @kolaps33 ?

Reply:
changed